linera-views/src/dynamo_db.rs

```diff
@@ -347,48 +380,63 @@ impl DynamoDbBatch {
         db: &DynamoDbClientInternal,
         base_key: &[u8],
     ) -> Result<JournalHeader, DynamoDbContextError> {
-        let delete_count = self.0.deletions.len();
```

Comment:
This function will need to be broken down at some point. Also, unit tests would help.

Reply:
Ok, but for future work.